gobblin-data-management/src/main/java/org/apache/gobblin/data/management/copy/iceberg/IcebergRegisterStep.java

```diff
@@ -64,6 +64,7 @@ public void execute() throws IOException {
     TableMetadata unusedNowCurrentDestMetadata = null;
     try {
       unusedNowCurrentDestMetadata = destIcebergTable.accessTableMetadata(); // probe... (first access could throw)
+      unusedNowCurrentDestMetadata.uuid(); // access to prevent findbugs "dead store to..." (Dodgy code Warning)
```

Comment:
Maybe log this out instead of a dead call if it's still a useful field?

Reply:
good call... added logging